### ISSUE TITLE
ntr: microtubule cortical anchor

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#13348]
Def The binding activity  that attaches a microtubule to the cell cortex
 

Descendent of 
GO:0008093 cytoskeletal adaptor activity
~GO:0043495 protein anchor~

Ref: PMID:25736293

**Comment by ValWood** [comment]
small def tweak:

current
protein anchor
Definition	Interacting selectively and non-covalently with both a protein or protein complex and a membrane, in order to maintain the localization of the protein at a specific location on the membrane.


" on the membrane" doesn't sound quite right for anchoring larger constructs..

Maybe at a "specific membrane location"is better?

**Comment by mah11** [comment]
I've looked briefly at the paper for the linked FYPO ticket, and I'm a little confused about membrane vs. cortex for the suggested term. The name seems to imply that the activity links a microtubule to the cell cortex, whereas the suggested link to 'protein anchor' would imply that the binding activity attaches the MT directly to the membrane.

**Comment by ValWood** [comment]
Yes the link to protein anchor was incorrect.

**Comment by hdrabkin** [comment]
done and done

**Comment by ValWood** [comment]
I'm reopening this one.

It seems strange that we have a MF term for "protein anchor" which is restricted to "anchoring to membrane" but we have "anchoring" process terms which apply to the cortex. It seems that we could use the MF term "protein anchor" to apply to gene products that function as protein anchors in  contexts other than  "anchoring to membrane".

Suggest that the term name of "protein anchor" is revised to "extrinsic membrane protein anchor" or something else specific to this particular context ?

**Comment by ValWood** [comment]
Hi Harold,

I don't see this term?

ntr: microtubule cortical anchor

Cheers,
Val

**Comment by hdrabkin** [comment]
I put this in 4/14; but you are right, it doesn't appear to be there. Let me try again.

**Comment by hdrabkin** [comment]
Hi Val
mictotubule cortical anchor activity is why; I have to fix the spelling

**Comment by hdrabkin** [comment]
we need a dictionary/spell check in Protege. sigh.
The id is GO:0106006; I checked in MGI; SO sorry. 
Typo fixed

**Comment by ValWood** [comment]
Not a problem ;)
